course-results/2020/Agshin-Balayev/day05_Genomic_Technologies.md

```diff
@@ -0,0 +1,88 @@
+uniparental disomy because under low heterozygosity, frequency of different alleles is low and more probes needed to detect variant allele.
+
+Limitations of Arrays:
+
+- Different probe density and per-rpobe speicificity which has induced creation of multiple algortihms with a lot of parameter optimization.
+
+- Assay sensitivity to detect short CNV and assumptions like diploid reference genome even if it is not the case. For example: duplication in reference genome causes more than two copies of one region.
+
+See ![aCGH method](https://www.researchgate.net/profile/Sebastiano_Cavallaro/publication/305078193/figure/fig2/AS:391208695287857@1470282829808/Array-CGH-procedure-is-characterized-by-the-isolation-of-DN$
+Advantages of Arrays: cost and throughput
+
+* Single Molecule Analysis:
+
+Immobilized genome is digested using restriction enzymes and based on ordering of portions is compared to in silico digested reference genome.This method allows to detect inversions, translocations, \
+copy number alterations and their locations.
+
+# Sequencing-based computational approach
+
+- NGS technologies seem to surpass arrays in the near future for CNV detection but are presented with computational and bioinformatic challenges. A lot of the computational algorithms have been \
+designed to investigate capillary sequencing reads and fully-sequenced large-insert clones.
+
+- Methods for computational analysis of CNVs through NGS technologies such whole-exome-sequencing and whole-genome-sequencing:
+
+* Read depth: method uses the deviation from Poisson distribution in mapping depth which at high level is an indicator of duplication and at low level of a deletion. But not all structural \
+variations can be identified using this technique. To discover smaller duplications and deletions at smaller breakpoint resolution **event-wise-testing algorithm** and **CVNator**  are used.
+
+* Read-pair technologies:
+
+- This can detect all types of strucutural variations. For instance, if paired-end reads are aligned further apart on the reference genome, it is deletion. If aligned closer on the reference genome, \
+it is insertion. If orientation of the reads are switched, then inversion or tandem duplication. Computational tools that use read-pair approach: PEMer, VariationHunter, BreakDancer, MoDIL, MoGUL, \
+HYDRA, Corona and SPANNER.
+
+* Split-read approach: approach uses the space between reads aligned to the reference genome. See the image below. Approach can detect all types of structural variations. And Pindel algorithm is an \
+example.
+
+
+* Assembly-based approach: assembly of reads into contigs are done through de novo assembly and local assembly techniques. Well-known de novo assembly algorithms for next-generation whole-genome \
+shotgun (NG-WGS) data include EULER-USR, ABySS, SOAPdenovo and ALLPATHS-LG. Using the Cortex assembler, variant assembly can be done entirely de novo or with different degrees of information from a \
```

Comment:
O.k: There is really a problem in copying text verbatim from articles. While in principle this is a "homework - do your self study about topic XXX" task, these tasks were also clearly mentioned as "to be looked at". So, definitely no copy / paste without clear quoting & source (Also, C & P should never be used for one's own private notes - doesn't help).

Please revise; amount of text & topical completeness is *les* important than demonstrated work w/ the topic.